tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+
+import great_expectations.expectations as gxe
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.test_utils.data_source_config import (
+    PandasDataFrameDatasourceTestConfig,
+)
+
+
+@parameterize_batch_for_data_sources(
+    data_source_configs=[
+        PandasDataFrameDatasourceTestConfig(),
+    ],
+    data=pd.DataFrame({"a": [1, 2]}),
+)
+def test_min(batch_for_datasource) -> None:
+    expectation = gxe.ExpectColumnMinToBeBetween(column="a", min_value=1, max_value=1)
+    result = batch_for_datasource.validate(expectation)
+    assert result.success
+
+
+@parameterize_batch_for_data_sources(
+    data_source_configs=[
+        PandasDataFrameDatasourceTestConfig(),
+    ],
+    data=pd.DataFrame({"a": [1, 2], "b": [3, 4]}),
+)
+def test_max(batch_for_datasource) -> None:
```

Comment:
Nit - could we expand this name to be `test_expect_column_max_to_be_between` or something more explicit in case we have other "max" related tests and this file grows significantly